lib/ain-ocean/src/indexer/poolswap.rs

```diff
@@ -184,7 +184,7 @@ fn create_new_bucket(
 
 impl IndexBlockStart for PoolSwap {
     fn index_block_start(self, services: &Arc<Services>, block: &BlockContext) -> Result<()> {
-        let mut pool_pairs = services.pool_pair_cache.get();
+        let mut pool_pairs =  ain_cpp_imports::get_pool_pairs();
```

Comment:
revert as cache doesn't work on array/vector..
todo: refine the flow

Reply:
[1206e36](https://github.com/DeFiCh/ain/pull/3107/commits/1206e36970071bfb2bc239482b6106f99ff82ca7)